AirCasting/SessionViews/ThresholdSettings/ThresholdsSettingsViewModel.swift

```diff
@@ -37,10 +37,16 @@ class ThresholdSettingsViewModel: ObservableObject {
             return
         }
         
-        let newValues: [Int32] = [thresholdVeryHigh, thresholdHigh, thresholdMedium, thresholdLow, thresholdVeryLow]
+        var newValues: [Int32] = [thresholdVeryHigh, thresholdHigh, thresholdMedium, thresholdLow, thresholdVeryLow]
             .map { formatter.value(from: $0) ?? 0 }
             .sorted { $0 < $1 }
         
+        // Prevents us from the situation when the user could change thresholds to have the same values
+        newValues = newValues.reversed().mapWithNext { lowerThreshold, higherThreshold in
```

Comment:
You are getting those elements here in reversed order actually, so it should be `higherThreshold, lowerThreshold`. It still works, but only because afterwards the values are sorted again. But this logic is a bit misleading and works kind of accidentally. :D @jankrzempek @pawelgil